waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -55,10 +57,9 @@ preCleanup :: AppSpec -> Path' Abs (Dir ProjectRootDir) -> IO ()
 preCleanup spec projectRootDir = do
   deleteGeneratedMigrationsDirIfRedundant spec projectRootDir
 
--- * Db generator
-
-genDb :: AppSpec -> [FileDraft]
-genDb spec = genPrismaSchema spec : maybeToList (genMigrationsDir spec)
+genDb :: AppSpec -> Generator [FileDraft]
+genDb spec =
+  genPrismaSchema spec <:> (maybeToList <$> genMigrationsDir spec)
```

Comment:
It is cool to see how, although we are in monad now and operating on monads, code looks so similar to the code before, thanks to the operators you wrote!

Reply:
Thanks! Yeah I think that turned out pretty nicely, thanks for the idea on adding `<:>`!